contracts/TestContracts/MassetManagerTester.sol

```diff
@@ -76,4 +122,8 @@ contract MassetManagerTester is IMassetManager {
 
         return _massetQuantity;
     }
+
+    function mAssetTokenTransferWithPermit() external view override returns(address) {
```

Comment:
should be a getter - see Masset review comments

Reply:
updated in this [commit](https://github.com/DistributedCollective/zero-contracts/pull/1/commits/12bbb6a0c7f6cef38929c6e2ce63905060be778e)